src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -1474,6 +1474,8 @@ func main() {
     // TODO: Add desired fields of the request body.
   }
 
+  // TODO: Use `Download` in place of `Do` below to download media content. For more information, see:
+  // https://godoc.org/google.golang.org/api/storage/v1
```

Comment:
Go programmers don't appear to like expository documentation ;-). I thought about a link to the method: are we comfortable assuming these URL anchors will remain stable?